src/build_vehicle_gui.cpp

```diff
@@ -1478,7 +1992,22 @@ struct BuildVehicleWindow : Window {
 	void SetStringParameters(int widget) const override
 	{
 		switch (widget) {
-			case WID_BV_CAPTION:
+		case WID_BV_SLOPE:
```

Comment:
Case labels should be indented a level deeper than the switch statement, so the code inside each case is indented two levels deeper than the switch statement.